koku/api/iam/sql/tenant-from-template0.sql

```diff
@@ -2353,8 +2353,6 @@ CREATE TABLE {{schema_name | sqlsafe}}."reporting_ocpawscostlineitem_project_dai
     "currency_code" character varying(10),
```

Comment:
Let's not make changes to this file. It is not used. If you want to remove the file instead, let's do that.